CMakeLists.txt

```diff
@@ -214,3 +237,13 @@ if(${GTest_FOUND})
 else()
     message("WARNING: Skipped stim_test target. `GTest` not found. To fix, follow Standalone CMake Project install instructions at https://github.com/google/googletest/blob/master/googletest/README.md")
 endif()
+
+find_package(Python COMPONENTS Interpreter Development)
+find_package(pybind11 CONFIG)
+if (${pybind11_FOUND} AND ${Python_FOUND})
+  pybind11_add_module(stim_python_bindings ${PYTHON_API_FILES} ${SOURCE_FILES_NO_MAIN})
+  set_target_properties(stim_python_bindings PROPERTIES OUTPUT_NAME stim)
+  add_compile_definitions(STIM_PYBIND11_MODULE_NAME=stim)
+else()
+  message("WARNING: Skipped stim_python_bindings target. `pybind11` not found")
```

Comment:
This should provide instructions on how to fix the issue, similar to how the test target failure links to the googletest documentation.

Reply:
done!